README.md

```diff
@@ -111,8 +111,8 @@ A p4c package is available in the following repositories for Ubuntu 20.04 and ne
 
 ```bash
 . /etc/os-release
```

Comment:
This should probably be lsb_release, will fix this later.

Reply:
yes
```
$ ls -al /etc/os-release 
lrwxrwxrwx. 1 root root 21 Mar 14 11:20 /etc/os-release -> ../usr/lib/os-release
```